### PR TITLE
test: add tests for release page, landing pages

### PR DESCRIPTION
**Pull request by HashimotoYT**
Added tests:

- Releases pages (responsive, and standard tests)
- Devtron and Spectron landing pages
- Buggy tests of responsiving language navigation bar.

Description for buggy tests section:

This section related to bugs whos confirmed, but not play big role. For give more information it can contain reproducible steps. It should get value what we should expect for verifying fixing.

### CODE REVIEW

File: cypress/integration/actions.js [hunk 1]
```diff
@@ -173,10 +183,63 @@ describe('electronjs.org', () => {
...
+    })
+
+    it('responsive navbar', () => {
+      cy.viewport('iphone-6', "portrait")
+      cy.visit(`${localhost}/releases/stable`)
+      cy.get('#release-navbar').should('have.class', 'd-none')
+      cy.get('.r-resp-header-toggle').should('be.visible').click()
+
+      // cy.get('#release-navbar').should('not.have.class', 'd-none')
+      // cy.get('#release-navbar > h3').should('be.visible').contains('Show Releases:')
```
Comment (BinaryMuse) [inline cypress/integration/actions.js]: Should we do something with these?

File: cypress/support/index.js [hunk 2]
```diff
@@ -18,3 +18,57 @@ import './commands'
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+// https://github.com/cypress-io/cypress/issues/1313#issuecomment-449456550
+// eslint-disable-next-line
+if (Cypress.env('RETRY_FAILED_TESTS')) {
```
Comment (BinaryMuse) [inline cypress/support/index.js]: I'm 👍 for this, but I'm curious what the reasoning was. Are some of the tests flakey?

File: cypress/integration/actions.js (hunk 1 above)
Comment (HashimotoYT) [reply cypress/integration/actions.js]: I'm trying to give chance for this today later if it's gone to be green all fine if red delete.

File: cypress/support/index.js (hunk 2 above)
Comment (HashimotoYT) [reply cypress/support/index.js]: Flakeys language bar and responsive nav bar tests.

**Pull request merged**